tests/Nox.Types.Tests/Types/NoxUri/NoxUriTests

```diff
@@ -0,0 +1,12 @@
+using FluentAssertions;
+namespace Nox.Types.Tests.Types;
+public class NoxUriTests
+{
+    [Fact]
+    public void NoxUri_ShouldBe_SameAs_SystemUri()
+    {
+        var expected = new Uri("http://www.regus.com");
+        var actual = NoxUri.From(expected);
+        actual.Value.Should().Be(expected);
+    }
+}
```

Comment:
the NoxTypesEntityFrameworkTests is not very well designed, one test does all, but you should have a test for this new type and validate how the EF handles it, and that no converter is needed

Reply:
done